presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -225,14 +227,13 @@ public JdbcOutputTableHandle beginInsertTable(ConnectorSession session, JdbcTabl
     public void finishInsertTable(ConnectorSession session, JdbcOutputTableHandle handle)
     {
         delegate.finishInsertTable(session, handle);
-        invalidateTablesCaches();
```

Comment:
This one line could go as separate commit. It is easy to not notice this, in this commit. It is logically different change at least to me. WDYT?

Reply:
Sure. I'll extract this